app/context.py

```diff
@@ -0,0 +1,41 @@
+import math
```

Comment:
Not sure why this file is showing up as having been added. Should already be in Development but I don't think it will cause any issues.

Reply:
I am honestly not sure either, it may have come from a rebase onto development?